index.js

```diff
@@ -19,46 +19,164 @@ const getCategory = (score, dementia) => {
 
 	if (
 		washingAndClothing &&
-		(timeAndSpace || dementia) &&
+		timeAndSpace &&
 		continence >= 3 &&
 		(transfers >= 3 || toilet >= 3 || meal >= 3)
 	) {
-		return 'CD';
+		return 'Cd';
 	}
 
 	if (washingAndClothing && transfers >= 3 && toilet >= 3 && (continence >= 3 || meal >= 3)) {
-		if (dementia) {
-			return 'CD';
-		}
-
 		return 'C';
 	}
 
 	if (washingAndClothing && (transfers >= 3 || toilet >= 3 || timeAndSpace)) {
-		if (dementia) {
-			return 'D';
-		}
-
 		return 'B';
 	}
 
-	if (washingOrClothing && !dementia) {
+	if (washingOrClothing) {
 		if (timeAndSpace) {
 			return 'B';
 		}
 
 		return 'A';
 	}
 
-	if (timeAndSpace && !dementia) {
+	if (timeAndSpace) {
 		return 'A';
 	}
 
-	if (dementia) {
-		return 'D';
+	return 'O';
+};
+
+const isIncontinent = score => {
+	const toilet = score.charAt(3);
+
+	return toilet >= 3;
+};
+
+const isIncontinentAndNeedHelpsWithTransferFoodOrToilet = score => {
+	// eslint-disable-next-line no-unused-vars
```

Comment:
Why disable eslint instead of simply removing the unused vars?

Reply:
![image](https://user-images.githubusercontent.com/1642668/126334190-2c9590ab-647b-415d-ba86-6c716661d5fa.png)